content/applications/sales/crm/pipeline/multi_sales_team.rst

```diff
@@ -1,47 +1,78 @@
+specific filters, such as country, language, etc. Set an invoicing target if your team has specific
+monthly revenue targets they need to achieve.
+
+.. image:: multiple_sales_teams/sales-team-creation.png
+   :align: center
+   :alt: Creating a Sales Team
+
+.. tip::
+   A team name, who you accept emails from, and a minimum score are all required form fields.
+
+Add members to your Sales Team
+------------------------------
+
+Team members can be added directly to the Sales Teams creation template when creating a new team or
+at a later time via editing. Click on *Add* under the *Assignment* tab, and choose a salesperson by
+typing in their name, selecting them from the drop down, or adding in a new name. If your team wants
+to ensure that your salespeople are not overdoing it, you can set a maximum number of leads in a 30
+day period.
+
+.. image:: multiple_sales_teams/add-a-salesperson.png
+   :align: center
+   :alt: Adding a Salesperson
+
+Assigning a *Team Leader* to your Sales Team follows a similar process as a adding a salesperson.
+Type in the leader's name or choose their name from the dropdown menu.
 
-There you can set an email alias to it. Every message sent to that email
-address will create a lead/opportunity.
+Both team members and Team Leaders can be added to multiple Sales Teams, which allows them to easily
+check all of the pipelines they need access to.
 
-.. image:: media/multi_sales_team01.png
-    :align: center
+Sales Team Overviews
+====================
 
-Add members to your sales channel
----------------------------------
+To head over to the overview, go to :menuselection:`CRM --> Configuration --> Sales Teams`. Any
+teams you are a part of will appear as overview tiles.
 
-You can add members to any channel; that way those members will see the
-pipeline structure of the sales channel when opening it. Any
-lead/opportunity assigned to them will link to the sales channel.
-Therefore, you can only be a member of one channel.
+Each tile shows:
 
-This will ease the process review of the team manager.
+- All of the open and overdue opportunities, quotations, and orders needing an invoice and the
+  revenue associated with each.
+- Graphs showing new opportunities broken down by week
+- An invoicing progress bar to show your team’s progress towards the invoicing goal set in the team
+  settings
 
-.. image:: media/multi_sales_team02.png
-    :align: center
+.. image:: multiple_sales_teams/sales-team-overview.png
+   :align: center
+   :alt: Sales Team Overview
 
-If you now filter on this specific channel in your pipeline, you will
-find all of its opportunities.
+Clicking on the three little dots opens another menu where you can quickly access information,
+such as viewing quotations, creating a new opportunity, or opening a report. Additionally, you can
+choose a color for your team as well as access the team configuration template.
```

Comment:
Editing this entire paragraph for brevity.

`Clicking on the three dots in the corner of a tile opens a navigational menu that lets you quickly view documents or reports, create new quotations or opportunities, pick a color for this team, or access its configuration page.`